apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/doedshendelse/kontrollpunkt/DoedshendelseKontrollpunktService.kt

```diff
@@ -347,15 +352,8 @@ class DoedshendelseKontrollpunktService(
             DoedshendelseKontrollpunkt.AnnenForelderIkkeFunnet
         }
 
-    private fun kontrollerEksisterendeSakEps(
-        doedshendelse: DoedshendelseInternal,
-        sak: Sak?,
-    ): DoedshendelseKontrollpunkt? {
-        if (doedshendelse.relasjon == Relasjon.EPS) {
-            sak?.let { return DoedshendelseKontrollpunkt.EpsHarSakIGjenny(sak) }
-        }
-        return null
-    }
+    private fun kontrollerEksisterendeSakEps(sak: Sak?): DoedshendelseKontrollpunkt? =
+        sak?.let { return DoedshendelseKontrollpunkt.EpsHarSakIGjenny(sak) }
```

Comment:
@sebassonav : Ser at vi for BP ser på iverksatt vedtak, mens for OMS kun ser på sak. Burde vi gjort dette likt?

Reply:
Høres fornuftig ut det!

Reply:
Oppretter en sak på det, det har noen funksjonelle problemstillinger også 👍